packages/documentation-site/patternfly-docs/patternfly-docs.config.js

```diff
@@ -1,34 +1,56 @@
 const componentsData = require('./pages/landing-pages/components/components-data.json');
-
-// This module is shared between NodeJS and babelled ES5
-module.exports = {
-  googleAnalyticsID: 'UA-47523816-6',
-  algolia: {
-    apiKey: 'a8fb1726b78594ff97a3418757514404',
-    appId: '79P4ZBH7A3',
-    indexName: 'patternfly'
-  },
-  hasGdprBanner: true,
-  hasFooter: true,
-  hasVersionSwitcher: true,
-  hasDesignGuidelines: true,
-  hasDarkThemeSwitcher: true,
-  componentsData,
-  sideNavItems: [
-    { section: 'get-started' },
-    { section: 'design-foundations' },
-    { section: 'accessibility' },
-    { section: 'UX writing' },
-    { section: 'components' },
-    { section: 'patterns' },
-    { section: 'extensions' }, 
-    { section: 'charts' },
-    { section: 'topology' },
-    { section: 'layouts' },  
-    { section: 'utility-classes' },
-    { section: 'developer-resources' },
-    { section: 'training' }, 
-  ],
-  topNavItems: [],
-  port: 8003,
+// This module is shared between NodeJS and babelled ES5, if this extension only build
+// exclude the other side nav items.
+if (process.env.EXTENSIONS_ONLY === 'true') { 
+    module.exports = {
+      googleAnalyticsID: 'UA-47523816-6',
```

Comment:
I'm not sure if we want to use the same analytics ID and algolia apiKey for the staging site. It'll be using all new URLs as well, so the search won't work. I wonder if you can take out a lot of these extra fields so that it more closely matches the one in core: https://github.com/patternfly/patternfly/blob/main/patternfly-docs/patternfly-docs.config.js

Reply:
Agreed here - the search returns results that would be hosted on an extensions-only site, most of the links in the footer aren't relevant (Components, Get started, etc pages won't exist on this site), and I'm not sure that the version switcher is relevant so we can likely strip this back further to keep the focus on the extensions.